TemplateFiles/DeploymentFiles/AddAztsFeatureConfiguraValuesTemplate.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "FeatureName": "CMET",
```

Comment:
Please rename all filenames having "Azts" to "AzTS"
Need some uniformity

Reply:
Done, renamed the file